Exercises-4-OOP/src/inheritanceanimal/Dog.java

```diff
@@ -0,0 +1,29 @@
+package inheritanceanimal;
+
+public class Dog extends Mammal {
+
+	public Dog() {
+
+	}
+
+	String greets = "woof";
+
+	public Dog(Mammal mammal1, String greets) {
+		super(mammal1.getName());
+		this.greets = greets;
+	}
+
+	public String getGreets() {
+		return greets;
+	}
+
+	public void setGreets(String greets) {
```

Comment:
This method is not part of the requirements, should be removed.

Reply:
if I remove it, the application won't run :(